docs/notebooks/ztf_bts-ngc.ipynb

```diff
@@ -0,0 +1,197 @@
+{
```

Comment:
<p>Can you add some links for NGC? I've never heard of it. Is it a real survey catalog, or just some fancy test data? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Adding wikipedia link - it is old enough</p>